src/gluonts/model/deepar/_network.py

```diff
@@ -15,6 +15,7 @@
 
 # Third-party imports
 import mxnet as mx
+import gluonnlp as nlp
```

Comment:
I don't see where we use gluonnlp in the file

Reply:
You are right. My mistake.
I was trying to use AR and TAR from gluonnlp, but ended up with implementing my own version, and forgot to delete the import.